### DISCUSSION TITLE
Communication channel

### DISCUSSION THREAD

**Discussion by GUKWAT** [ryansurf/cli-surf#11] [Q&A] [answered]
is there a discord channel or teams group for this repo?

**Comment by ryansurf** [comment, marked as answer]
Discord link [here](https://discord.com/invite/He2UpxRuJP)

**Comment by GUKWAT** [comment]
Hey @ryansurf, have you created the discord channel yet?

**Reply by ryansurf** [reply]
[Here](https://discord.gg/He2UpxRuJP) is the invite link!

**Reply by GUKWAT** [reply]
Thank you!
